Sources/BraveWallet/Panels/Signature Request/SignMessageRequestView.swift

```diff
@@ -0,0 +1,400 @@
+            copy.replaceSubrange(range, with: "\n\(uuid.uuidString) <\(newlines.count)>\n")
+          }
+        }
+      }
+      if message.hasUnknownUnicode {
+        result = result.printableWithUnknownUnicode
+      }
+
+      return result.replacingOccurrences(of: uuid.uuidString, with: "\u{00B6}")
+    }
+  }
+  
+  /// Header containing the current requests network chain name, and a `1 of N` & `Next` button when there are multiple requests.
+  private var requestsHeader: some View {
+    HStack {
+      if let network {
+        Text(network.chainName)
+          .font(.callout)
+          .foregroundColor(Color(.braveLabel))
+      }
+      Spacer()
+      if requestCount > 1 {
+        NextIndexButton(
+          currentIndex: requestIndex,
+          count: requestCount,
+          nextTapped: nextTapped
+        )
+      }
+    }
+  }
+  
+  private var accountInfoAndOrigin: some View {
+    VStack(spacing: 8) {
+      Blockie(address: account.address)
+        .frame(width: min(blockieSize, maxBlockieSize), height: min(blockieSize, maxBlockieSize))
+      AddressView(address: account.address) {
+        VStack(spacing: 4) {
+          Text(account.name)
+            .font(.subheadline.weight(.semibold))
+            .foregroundColor(Color(.braveLabel))
+          Text(account.address.truncatedAddress)
+            .font(.subheadline.weight(.semibold))
+            .foregroundColor(Color(.secondaryBraveLabel))
+        }
+      }
+      Text(originInfo: request.originInfo)
+        .font(.caption)
+        .foregroundColor(Color(.braveLabel))
+        .multilineTextAlignment(.center)
+    }
+    .accessibilityElement(children: .combine)
+  }
+  
+  var body: some View {
+    ScrollView {
+      VStack {
+        requestsHeader
+        
+        VStack(spacing: 12) {
+          accountInfoAndOrigin
+          
+          Text(Strings.Wallet.signatureRequestSubtitle)
+            .font(.headline)
+            .foregroundColor(Color(.bravePrimary))
+          
+          if needPilcrowFormatted[request.id] == true || currentRequestMessage?.hasUnknownUnicode == true {
+            MessageWarningView(
+              needsPilcrowFormatted: needPilcrowFormatted[request.id] == true,
+              hasUnknownUnicode: currentRequestMessage?.hasUnknownUnicode == true,
+              isShowingOriginalMessage: showOrignalMessage[request.id] == true,
+              action: {
+                let value = showOrignalMessage[request.id] ?? false
+                showOrignalMessage[request.id] = !value
+              }
+            )
+          }
+        }
+        .padding(.vertical, 32)
+        StaticTextView(text: requestDisplayText, attributedText: requestDisplayAttributedText, isMonospaced: false)
+          .frame(maxWidth: .infinity)
+          .frame(height: staticTextViewHeight)
+          .background(Color(.tertiaryBraveGroupedBackground))
+          .clipShape(RoundedRectangle(cornerRadius: 5, style: .continuous))
```

Comment:
May be able to get away with `.background(Color(.tertiaryBraveGroupedBackground), in: RoundedRectangle(cornerRadius: 10, style: .continuous))`